src/Tizen.TV.UIControls.Forms/ContentCreator.cs

```diff
@@ -0,0 +1,126 @@
+﻿using System;
+using Xamarin.Forms;
+
+namespace Tizen.TV.UIControls.Forms
+{
+    public class ContentCreator
+    {
+        public static View CreateContent(DataTemplate template, Object data, double heightRequest, double widthRequest)
+        {
+            if (template != null)
+            {
+                var content = template.CreateContent();
+                if (content is View view)
+                    return view;
+                else if (content is ViewCell viewCell)
+                    return viewCell.View;
+                else if (content is ImageCell imageCell)
+                    return CreateContent(imageCell, heightRequest, widthRequest);
+                else if (content is TextCell textCell)
+                    return CreateContent(textCell);
+            }
+            return CreateContent(new TextCell { Text = data?.ToString() });
+        }
+
+        static View CreateContent(ImageCell cell, double heightRequest, double widthRequest)

```

Comment:
`double heightRequest` `double widthRequest` are not used
```suggestion
        static View CreateContent(ImageCell cell)

```